torch/csrc/jit/codegen/cuda/kernel_ir_printer.cpp

```diff
@@ -304,6 +304,10 @@ void IrPrinter::visit(const kir::BroadcastOp* node) {
   indent() << gen(node->out()) << " = BROADCAST(" << use(node->in()) << ")\n";
 }
 
+void IrPrinter::visit(const kir::TransposeOp* node) {
+  indent() << gen(node->out()) << " = TRANSPOSE(" << use(node->in()) << ")\n";
```

Comment:
same as in the fusion ir printer: don't we want to print the details on the transpose?

Reply:
I think it's okay without it. `BroadcastOp` doesn't do so either.